extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -227,6 +262,16 @@ export class Visibility {
   }
 
   /** @private */
+  visibilityListener_() {
+    // viewer.getVisibilityState works when the page is loaded inside a viewer.
```

Comment:
it wasn't working at some point and I had more code in this function. since that code is removed, this comment did not make sense. removed.